frontend/packages/dev-console/integration-tests/support/pageObjects/addHealthChecks-po.ts

```diff
@@ -37,4 +37,11 @@ export const addHealthChecksObj = {
   },
   add: '[data-test-id="submit-button"]',
   cancel: '[data-test-id="reset-button"]',
+  save: '[data-test-id="submit-button"]',
+  healthChecksForm: 'div.odc-heath-check-probe-form',
+  successText: 'span.odc-heath-check-probe__successText',
+  removeReadinessProbeIcon: '',
+  readinessProbeAdded: '',
+  livenessProbeAdded: '',
+  addStartUpProbeAdded: '',
```

Comment:
will these be `''` or we need to add selector?

Reply:
Yes, these are placeholders for selectors, currently I didn't find any stable locator. So left it as blank.